authorize_response.go

```diff
@@ -54,24 +52,13 @@ func (a *AuthorizeResponse) AddHeader(key, value string) {
 	a.Header.Add(key, value)
 }
 
-func (a *AuthorizeResponse) GetQuery() url.Values {
-	return a.Query
+func (a *AuthorizeResponse) GetParameters() url.Values {
+	return a.Parameters
 }
 
-func (a *AuthorizeResponse) GetFragment() url.Values {
```

Comment:
I like this cleanup. It was always strange to me that callers had to know where should they put parameters (fragment or query).